content/posts/interview_template_AndersNord_2023-02-05.md

```diff
@@ -0,0 +1,56 @@
+title: Intervju med medlem: Anders Nord
+date: 2023-02-05
+modified:
+author:
+category: Interviews
+tags: interview, climate, ifk
+data:
+status: published
+
+![Bild av intervjuobjekt]({{post_url}}/data/INTERVJUBILD)
```

Comment:
Send a picture over and I can put it on our S3 like storage.

Reply:
What do you want to do about the image?

Reply:
Put it in, I suppose. Could you add the image to the article about ifk-smhi too? Would be awesome!

Reply:
```suggestion
![Anders Nord]({{post_url}}/data/anders_nord.jpeg)
```